source/templates.tex

```diff
@@ -1328,6 +1337,10 @@
 
 const char p[] = "Vivisectionist";
 X<int,p> x2;                    // OK
+
+class A { constexpr A(const char*) {} };
```

Comment:
This example is wrong. Add a defaulted `operator<=>` here to fix it.

Reply:
The example is even more wrong: The type of the second template-parameter of X is still "const char *" and does not involve "A" at all.